docs/docs/started/installing-nebari.md

```diff
@@ -0,0 +1,47 @@
+---
+id: installing-nebari
+---
+
+# Installing Nebari
+
+A guide to help you install Nebari for your project.
+
+This installation guide provides the basic needs to install and deploy Nebari for the first time and assumes the user is already familiar with the [Conda](https://docs.conda.io/projects/conda/en/latest/) and [Python packaging](https://packaging.python.org/en/latest/tutorials/installing-packages/#installing-packages) ecosystems. If you are already familiar to Nebari and would like to check the advance configuration, feel free to skip to "advanced-settings" section.
```

Comment:
What is the advanced-setting section? Is it a future document?

Reply:
yup, future document. Just commented in there because: